core/src/actions/properties.ts

```diff
@@ -108,7 +108,7 @@ export class PropertiesList extends AuthenticatedAction {
       }
     }
 
-    const total = await Property.scope(null).count();
```

Comment:
Sorry, I should have been more clear in my suggestion. The tests are failing because `findAndCountAll` returns an object with the rows and the count, so you can be sure the total count refers to whatever you're looking up. My suggestion was to replace `await Property.scope(null).findAll({` on line 71 with `await Property.scope(null).findAndCountAll({` - here's an example on the `exports` action:

https://github.com/grouparoo/grouparoo/blob/c7bce3cac78f48bfc6ace66da041c4c538a95257/core/src/actions/exports.ts#L45

Reply:
This works, but now the data is being fetched twice. 

I'm referring to removing this line from here completely and making a change higher up:

https://github.com/grouparoo/grouparoo/blob/041e196e850bf83d12ff6012dfd555d9437b99c0/core/src/actions/properties.ts#L70-L80

This would instead be something like:

```ts
    const { rows, count: total } = await Property.scope(null).findAndCountAll({ 
        include, 
        limit: params.limit, 
        offset: params.offset, 
        order: params.order, 
        where, 
    });

    const properties = rows.filter((p) => 
	params.modelId ? p.source?.modelId === params.modelId : true 
   ); 
```
That way we're grabbing the total and finding in the same call, guaranteeing the same args.
Let me know if you want to have a quick chat

Reply:
Oh of course. 🤦🏼  Despite the name of the function being super clear about what it didn't I didn't think too much into it when I swapped it.